src/main/java/org/logstash/netty/SslContextBuilder.java

```diff
@@ -68,7 +70,7 @@
     private String[] supportedCiphers = ((SSLServerSocketFactory)SSLServerSocketFactory
             .getDefault()).getSupportedCipherSuites();
     private String[] ciphers = DEFAULT_CIPHERS;
-    private String[] protocols = new String[] { "TLSv1.2" };
+    private String[] protocols = new String[] { "TLSv1.3" };
```

Comment:
for backwards compatibility v1.3 should not be forced, having `new String[] { "TLSv1.2", "TLSv1.3" };` should provide backwards compatibility with enabling TLS 1.3 as well

Reply:
Fixed.